src/wp-includes/class-wp-block-bindings-source.php

```diff
@@ -75,14 +74,25 @@ public function __construct( string $name, array $source_properties ) {
 	 *
 	 * @since 6.5.0
 	 *
+	 * This function calls the callback function specified in the `$get_value_callback` property
+	 * with the given arguments and returns the result. It then applies the filter
+	 * `block_bindings_source_value` to the value before returning it.
+	 *
 	 * @param array    $source_args     Array containing source arguments used to look up the override value, i.e. {"key": "foo"}.
 	 * @param WP_Block $block_instance  The block instance.
 	 * @param string   $attribute_name  The name of the target attribute.
-	 *
 	 * @return mixed The value of the source.
 	 */
 	public function get_value( array $source_args, $block_instance, string $attribute_name ) {
-		return call_user_func_array( $this->get_value_callback, array( $source_args, $block_instance, $attribute_name ) );
+		/**
+		 * Filters the output of a block bindings source.
+		 *
+		 * @since 6.7.0
+		 *
+		 * @param mixed $value Value returned by `get_value_callback` after applying the filter.
+		 */
+		$value = call_user_func_array( $this->get_value_callback, array( $source_args, $block_instance, $attribute_name ) );
+		return apply_filters( 'block_bindings_source_value', $value, $this->name, $source_args, $block_instance, $attribute_name );
```

Comment:
It still needs some further refinements, so it gets correctly recognized by the parser that generates the documentation for all filters:
```suggestion
		$value = call_user_func_array( $this->get_value_callback, array( $source_args, $block_instance, $attribute_name ) );
		/**
		 * Filters the value of a block bindings source.
		 *
		 * @since 6.7.0
		 *
		 * @param mixed        $value                 The computed value for the source.
		 * @param string        $name                 The name of the source.
		 * @param array         $source_args      Array containing source arguments used to look up the override value, i.e. { "key": "foo" }.
		 * @param WP_Block $block_instance The block instance.
		 * @param string        $attribute_name The name of an attribute.
		 */
		return apply_filters( 'block_bindings_source_value', $value, $this->name, $source_args, $block_instance, $attribute_name );
```